Paws4Me/Paws4Me/Display Single Pet Details/View Controllers/PetSingleDetailsViewController.swift

```diff
@@ -24,16 +25,26 @@ class PetSingleDetailsViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         setPlaceholderImage()
+        adoptPetButton.addCornerRadius()
+        saveSinglePetButton.changeBorderLook()
+        saveSinglePetButton.addCornerRadius()
         updateUI()
     }
 
     override func viewWillAppear(_ animated: Bool) {
-        isPetSaved()
+        let isSaved = isPetSaved()
+        if isSaved {
```

Comment:
So no need for separate `func` 

`if singlePetViewModel.isPetSaved() {
`

Reply:
Agreed 👍🏻 much more simple